<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AddSensor</class>
 <widget class="QDialog" name="AddSensor">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>466</width>
    <height>200</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="maximumSize">
   <size>
    <width>466</width>
    <height>200</height>
   </size>
  </property>
  <property name="baseSize">
   <size>
    <width>466</width>
    <height>200</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="locale">
   <locale language="English" country="UnitedStates"/>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout_2">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout" stretch="0">
     <item>
      <layout class="QVBoxLayout" name="verticalLayout" stretch="0,0,0,0,0">
       <item>
        <layout class="QHBoxLayout" name="nameLayout">
         <item>
          <widget class="QLabel" name="NameLabel">
           <property name="text">
            <string>Name -</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="NameInput">
           <property name="inputMask">
            <string/>
           </property>
           <property name="frame">
            <bool>false</bool>
           </property>
           <property name="placeholderText">
            <string>ex. &quot;Worst CPU Temperature&quot;</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="descLayout">
         <item>
          <widget class="QLabel" name="DescLabel">
           <property name="text">
            <string>Description -</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="DescInput">
           <property name="frame">
            <bool>false</bool>
           </property>
           <property name="placeholderText">
            <string>ex. &quot;Matches with a Ryzen 5 7600X CPU&quot;</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="typeLayout">
         <item>
          <widget class="QLabel" name="TypeLabel">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="maximumSize">
            <size>
             <width>100</width>
             <height>16777215</height>
            </size>
           </property>
           <property name="text">
            <string>Component -</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QRadioButton" name="CPUButton">
           <property name="text">
            <string>CPU</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QRadioButton" name="GPUButton">
           <property name="text">
            <string>GPU</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QRadioButton" name="RAMButton">
           <property name="text">
            <string>RAM</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="boundsLayout">
         <item>
          <spacer name="horizontalSpacer_2">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <layout class="QHBoxLayout" name="maxLayout">
           <property name="spacing">
            <number>1</number>
           </property>
           <item>
            <widget class="QLabel" name="MaxLabel">
             <property name="minimumSize">
              <size>
               <width>30</width>
               <height>0</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>16777215</width>
               <height>16777215</height>
              </size>
             </property>
             <property name="text">
              <string>Max - </string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="MaxInput">
             <property name="maximumSize">
              <size>
               <width>75</width>
               <height>16777215</height>
              </size>
             </property>
             <property name="frame">
              <bool>false</bool>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <spacer name="horizontalSpacer">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <layout class="QHBoxLayout" name="tempLayout">
           <property name="spacing">
            <number>1</number>
           </property>
           <item>
            <widget class="QLabel" name="TempLabel">
             <property name="minimumSize">
              <size>
               <width>30</width>
               <height>0</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>16777215</width>
               <height>16777215</height>
              </size>
             </property>
             <property name="text">
              <string>Temp - </string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="lineEdit">
             <property name="maximumSize">
              <size>
               <width>75</width>
               <height>16777215</height>
              </size>
             </property>
             <property name="frame">
              <bool>false</bool>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <spacer name="horizontalSpacer_3">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QPushButton" name="NewSensorButton">
         <property name="text">
          <string>Create</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
